### PR TITLE
Fixed DAO return type

### PR DESCRIPTION
**Pull request by IngvarJackal**
Fixed type for #289 . No localization needed.

### CODE REVIEW

File: src/main/scala/ru/org/codingteam/horta/plugins/karma/KarmaRepository.scala [hunk 1]
```diff
@@ -16,11 +16,11 @@ case class KarmaRepository(session: DBSession) {
        """.map(rs => rs.string("member") + " " + rs.string("karma")).list().apply()
   }
 
-  def getKarma(room: String, member: String): Option[Int] = {
+  def getKarma(room: String, member: String): Int = {
```
Comment (rexim) [inline src/main/scala/ru/org/codingteam/horta/plugins/karma/KarmaRepository.scala]: Doesn't this change require to modify places where the method is used?

Comment (IngvarJackal) [inline src/main/scala/ru/org/codingteam/horta/plugins/karma/KarmaRepository.scala]: Of course not. Everything will be fine.

Comment (IngvarJackal) [inline src/main/scala/ru/org/codingteam/horta/plugins/karma/KarmaRepository.scala]: Mr. @ForNeVeR has changed relevant code already.

**Comment by ForNeVeR** [comment]
Okay, thanks!

**Pull request merged**